Certificate issuance error (does not accept fresh agreement)

<!--
--> Starting operation: Reconcile certificates for: delta.angelka.ru, mta-sts.delta.angelka.ru, www.delta.angelka.ru 
    [delta.angelka.ru] 20260607205639 [DEBUG] acmetool.storageops: Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): best certificate satisfying is <nil>, err=Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): no certificate satisfies this target
    [delta.angelka.ru] 20260607205639 [DEBUG] acmetool.storageops: Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): requesting certificate
    [delta.angelka.ru] 20260607205640 [INFO] acme.interactor: interaction auto-responder couldn't give a canned response: unknown unique ID, cannot respond: "acme-agreement:https://letsencrypt.org/documents/LE-SA-v1.7-June-04-2026.pdf"
    [delta.angelka.ru] 20260607205640 [ERROR] acmetool.storageops: Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): failed to request certificate: cannot prompt the user: currently non-interactive; try running without --batch flag
    [delta.angelka.ru] 20260607205640 [DEBUG] acmetool.storageops: done processing targets, reconciliation complete, 1 errors occurred
    [delta.angelka.ru] 20260607205640 [ERROR] acmetool.storageops: error while processing targets: the following errors occurred:
    [delta.angelka.ru] error satisfying Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): cannot prompt the user: currently non-interactive; try running without --batch flag
    [delta.angelka.ru] 20260607205640 [ERROR] acmetool.storageops: failed to reconcile: the following errors occurred:
    [delta.angelka.ru] error satisfying Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): cannot prompt the user: currently non-interactive; try running without --batch flag
    [delta.angelka.ru] 20260607205640 [DEBUG] acmetool.storageops: disjoint hostname mapping: "delta.angelka.ru" -> Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0)
    [delta.angelka.ru] 20260607205640 [DEBUG] acmetool.storageops: disjoint hostname mapping: "mta-sts.delta.angelka.ru" -> Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0)
    [delta.angelka.ru] 20260607205640 [DEBUG] acmetool.storageops: disjoint hostname mapping: "www.delta.angelka.ru" -> Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0)
    [delta.angelka.ru] 20260607205640 [DEBUG] acmetool.storageops: could not find certificate satisfying Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): no certificate satisfies this target
    [delta.angelka.ru] 20260607205640 [DEBUG] acmetool.storageops: could not find certificate satisfying Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): no certificate satisfies this target
    [delta.angelka.ru] 20260607205640 [DEBUG] acmetool.storageops: could not find certificate satisfying Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): no certificate satisfies this target
    [delta.angelka.ru] 20260607205640 [CRITICAL] acmetool: fatal: reconcile: the following errors occurred:
    [delta.angelka.ru] error satisfying Target(delta.angelka.ru,mta-sts.delta.angelka.ru,www.delta.angelka.ru;https://acme-v02.api.letsencrypt.org/directory;0): cannot prompt the user: currently non-interactive; try running without --batch flag

-->

- Server OS (Operating System) - preferably Debian 12: 
- On which OS you run cmdeploy: scripts/cmdeploy run  
- chatmail/relay version: `❯ git rev-parse HEAD
6def189d16143978b004c18e38d7ae4b3aa474ac
`

## Expected behavior Successful installation

*What did you try to achieve?* Correct installation

## Actual behavior Error

*What happened instead?*

### Steps to reproduce the problem:

1. 
2.

### Screenshots

### Logs

--- a/cmdeploy/src/cmdeploy/acmetool/response-file.yaml.j2
+++ b/cmdeploy/src/cmdeploy/acmetool/response-file.yaml.j2
@@ -1,2 +1,2 @@
 "acme-enter-email": "{{ email }}"
-"acme-agreement:https://letsencrypt.org/documents/LE-SA-v1.6-August-18-2025.pdf": true
+"acme-agreement:https://letsencrypt.org/documents/LE-SA-v1.7-June-04-2026.pdf": true

